src/svg/podcast.svg

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
-    <path d="M16 0C7.1.1-.1 7.3 0 16.1.1 24.9 7.2 32 16 32h.1c4.3 0 8.3-1.7 11.3-4.8s4.6-7 4.6-11.3C31.9 7.1 24.8 0 16 0zm9.8 25.7c-2.6 2.6-6.1 4.1-9.8 4.1-7.6 0-13.8-6.2-13.8-13.8C2.2 8.4 8.3 2.1 16 2.1c7.7 0 13.9 6.2 13.9 13.8 0 3.7-1.5 7.2-4.1 9.8z"/>
-    <path d="M20.3 11.1c-.6 0-1.1.5-1.1 1.1v6.2c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1v-6.2c0-.6-.5-1.1-1.1-1.1zM24.6 8.7c-.6 0-1.1.5-1.1 1.1v11.5c0 .6.5 1.1 1.1 1.1.6 0 1.1-.5 1.1-1.1V9.8c0-.6-.5-1.1-1.1-1.1zM16 5c-.6 0-1.1.5-1.1 1.1V26c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1V6c0-.5-.5-1-1.1-1zM11.7 8.3c-.6 0-1.1.5-1.1 1.1v12.5c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1V9.4c0-.6-.5-1.1-1.1-1.1zM7.4 12.3c-.6 0-1.1.5-1.1 1.1v3.5c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1v-3.5c0-.6-.5-1.1-1.1-1.1z"/>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 34 28" xml:space="preserve">
```

Comment:
(Almost) Every svg on the project are formated to 32x32 could you please scale this one too?